Improve the readme (#35691)
nextcloud/server: README.md

[![codecov](https://codecov.io/gh/nextcloud/server/branch/master/graph/badge.svg)](https://codecov.io/gh/nextcloud/server)
[![CII Best Practices](https://bestpractices.coreinfrastructure.org/projects/209/badge)](https://bestpractices.coreinfrastructure.org/projects/209)

> **A safe home for all of your data.**

Reviewer: ```suggestion
**Regain control over your data**
```
Let's use the slogan from our front page, not this super old one! Fresh and tasty.

* 📁 **Access your Data** You can store your files, contacts, calendars, and more on a server of your choosing.
* 🔄 **Sync your Data** You keep your files, contacts, calendars, and more synchronized amongst your devices.
* 🙌 **Share your Data** …by giving others access to the stuff you want them to see or to collaborate with.
* 🚀 **Expandable with hundreds of Apps** ...like [Calendar](https://github.com/nextcloud/calendar), [Contacts](https://github.com/nextcloud/contacts), [Mail](https://github.com/nextcloud/mail), [Video Chat](https://github.com/nextcloud/spreed) and all those you can discover in our [App Store](https://apps.nextcloud.com).
* 🔒 **Security** with our encryption mechanisms, [HackerOne bounty program](https://hackerone.com/nextcloud) and two-factor authentication.

> Do you want to learn more about how you can use Nextcloud to access, share and protect your files, calendars, contacts, communication and more at home and/or in your organization? [**Learn about all our Features**](https://nextcloud.com/athome/).

Reviewer: ```suggestion
Do you want to learn more about how you can use Nextcloud to access, share and protect your files, calendars, contacts, communication and more at home and/or in your organization? [**Learn about all our Features**](https://nextcloud.com/hub/).
```
Good, but we should also update that link - athome is gone, let's just point to the product page - hub!

## Get your Nextcloud 🚚

> - ☑️ [**Simply sign up**](https://nextcloud.com/signup/) with one of our providers either through our website or through our apps directly.

Reviewer: No, the change is correct, it's OUR apps (android, iOS, desktop) through which you can sign up with a third party provider ;-) So this is actually good.

> - ☑️ [**Simply sign up**](https://nextcloud.com/signup/) with one of our providers either through our website or through our apps directly.

Author: ```suggestion 
☑️ [**Simply sign up**](https://nextcloud.com/signup/) with one of our providers either through our website or through apps (Android, iOS, Desktop) directly.
```

would this be good then? @jospoortvliet

> - 🖥 [**Install**](https://nextcloud.com/install/#instructions-server) a server by yourself on your hardware.

Reviewer: Yeah, I think thats fine honestly. Everything is appliances these days, ppl expect a docker. And we deliver :D

> - 📦 Buy one of our awesome [**devices** ](https://nextcloud.com/devices/) with Nextcloud preinstalled.

Author: Oh my bad. then does this seem fair to change to?
```suggestion
- 📦 Buy one of the awesome [**devices listed here**](https://nextcloud.com/devices/) with Nextcloud preinstalled.
```

- 🏢 Find a [**service provider**](https://nextcloud.com/providers/) who can host Nextcloud for you or for your company.

Are you in the Private Sector, Public Sector, or Education? You may want to have a look into [**Nextcloud Enterprise**](https://nextcloud.com/enterprise/) provided by Nextcloud GmbH.

## Get in touch 💬

> - [📋 Forum](https://help.nextcloud.com)
> - [👥 Facebook](https://www.facebook.com/nextclouders)
> - [🐣 Twitter](https://twitter.com/Nextclouders)
> - [🐘 Mastodon](https://mastodon.xyz/@nextcloud)

Author: This is valid, they are both valid. i guess i just like it more. it's personal choice. if you don't like it that is ok. we can keep it the original. 
```suggestion
* [📋 Forum](https://help.nextcloud.com)
* [👥 Facebook](https://www.facebook.com/nextclouders)
* [🐣 Twitter](https://twitter.com/Nextclouders)
* [🐘 Mastodon](https://mastodon.xyz/@nextcloud)
```

> You can also [get support from Nextcloud](https://nextcloud.com/support)!

Author: My thoughts for this change was, since the link takes you to the support page of Nextcloud, it is support from next cloud. the supporting documents linked on the page are from the next cloud community. It's semantically indifferent to me i just thought it made sense that the help was coming from the community. What are your thoughts?

> You can also [get support from Nextcloud](https://nextcloud.com/support)!

Reviewer: I agree with Come it's a little confusing. You can get support from multiple sources, from Nextcloud GmbH to Nextcloud-community-members or, well, your neighboor or something ;-)

But it's support for Nextcloud-the-product in all cases. So I'd rather say that.

## Join the team 👪

> There are many ways to contribute. Find out [how to get involved](https://nextcloud.com/contribute/), 
> - Translator
> - Designer
> - Tester
> - Community support
> 
> and much more... 😍

Author: This is valid. I guess i just like it more. it's personal choice. if you don't like it that is ok. we can keep it the original.
```suggestion
There are many ways to contribute, of which development is only one! Find out [how to get involved](https://nextcloud.com/contribute/). This includes a translator, designer, tester, helping others, and much more! 😍
```
I did change it a little to make it easier to read.

### Development setup 👩‍💻

1. 🚀 [Set up your local development environment.](https://docs.nextcloud.com/server/latest/developer_manual/getting_started/devenv.html)
2. 🐛 [Pick a good first issue.](https://github.com/nextcloud/server/labels/good%20first%20issue)
3. 👩‍🔧 Create a branch and make your changes. Remember to sign off your commits using `git commit -sm "Your commit message"`.
4. ⬆ Create a [pull request](https://opensource.guide/how-to-contribute/#opening-a-pull-request) and `@mention` the people from the issue to review.
5. 👍 Fix things that come up during a review.
6. 🎉 Wait for it to get merged!

> Third-party components are handled as git submodules which have to be initialized first. Git checkout invoking `git submodule update --init` or a similar command is needed, for details see [Git documentation](https://git-scm.com/docs/user-manual).

Reviewer 2: The sentence lost its meaning, it was only missing a comma after «invoking»

Author: I can see your point.
```suggestion
Third-party components are handled as git submodules which have to be initialized first. So aside from the regular git checkout invoking, `git submodule update --init` or a similar command is needed. For details see [Git documentation](https://git-scm.com/docs/user-manual).
```
Do you agree to adding a link to the git documentation?

Reviewer: But in what you propose, you put the comma wrong. People need to invoke the command 'git submodule' etc, so the coma should be after 'checkout', not after invoking:

> So aside from the regular git checkout, invoking `git submodule update --init` or a similar command is needed. For details see [Git documentation](https://git-scm.com/docs/user-manual)

Author: ```suggestion
Third-party components are handled as git submodules which have to be initialized first. So aside from the regular git checkout, invoking `git submodule update --init` or a similar command is needed. For details see [Git documentation](https://git-scm.com/docs/user-manual).
```

so like this @jospoortvliet?
